### ISSUE TITLE
LMS integration managing student accounts which were created with the wrong email address

### ISSUE THREAD

**Issue by pearcej** [RunestoneInteractive/InstructorGuide#23]
**What Course are you in**
InstructorGuide (well, at least this is where the solution to the problem needs to live.)

**What is your username**
pearcej
Please note that I plan to issue a pull request soon in order to help others solve this problem.

**Describe the bug**
Sometimes students who are in a course that is using an LMS with LTI integration already have a Runestone account made with with an email address that does not match the email address needed by the LMS. This can happen, for example, when a student previously used Runestone books with another instructor who did not use LTI integration.

**Proposed Solution**
1. The instructor needs to log into Runestone via clicking an assignment link in your LMS. This will ensure you are using an LMS-attached instructor’s account. 
2. Go to the Instructor’s page by clicking the person icon. Then go to your gradebook. It should be sorted by last name. Hopefully, for each of these students you will see two accounts that are right next to each other. One of these will likely have 0% on an assignment and the other will likely have credit. Note the email address and UName of the account with 0%. For a big class, it might be easiest to download the gradebook for this, so you can look at it in Excel. 
3. Note the grades of the students who have two accounts and manually fix the gradebook in the LMS so the correct grade is showing.
4. Next, click the Admin tab and go to Manage Students. Then carefully remove each of the students with the UName and email address that got 0 credit. 

Note: Since I have done the above for my three students who had problems several weeks ago, LTI has worked correctly for all of my students.

I was thinking of putting this information at the end of section 4.3 Integrating with your LMS or possibly as a separate 4.4 section. Please let me know which makes more sense.

**Comment by bnmnetp** [comment]
I would be good to emphasize that when you remove a student from your course you are not deleting that student from the system.  That students account will continue to exist.  They will be un-enrolled from your course and moved to the open course.

This too can cause confusion if said student comes and logs into runestone outside of the LMS.

**Comment by bnmnetp** [comment]
The email does not match, but the username does match, is that the root of what causes this?

**Comment by pearcej** [comment]
Not exactly. What happens in my experience with Moodle is if the email address does not match exactly what is needed by the LMS is a new account is made with the correct email address and the new username is the same as the email address.

**Comment by bnmnetp** [comment]
I think different LMS are different.  Canvas does not seem to use the email as the username.  Maybe its something the admin configures.

**Comment by pearcej** [comment]
Interesting. Given these issues, it make make sense to stress that an instructor should be consistent in either using LTI for all assignments or not using LTI at all for a given course...

**Comment by bnmnetp** [comment]
Yes, we pretty much enforce that way of thinking as if you have LTI configured and you go to the assignment page, it tells you you have to launch any assignment through the LMS.
